tests/trestle/core/commands/assemble_test.py

```diff
@@ -65,7 +65,7 @@ def test_assemble_catalog(testdata_dir: pathlib.Path, tmp_trestle_dir: pathlib.P
         assert rc == 0
 
     # Read assembled model
-    actual_model = Catalog.oscal_read(pathlib.Path('dist/catalog.json'))
+    actual_model = Catalog.oscal_read(pathlib.Path('dist/catalogs/mycatalog.json'))
     _, _, expected_model = load_distributed(mycatalog_dir / 'catalog.json')
 
```

Comment:
I would add a test on `assessment-results` to deal with the case mentioned below.